common/lib/xmodule/xmodule/js/src/video/07_video_volume_control.js

```diff
@@ -37,35 +37,33 @@ function() {
         /** Step to increase/decrease volume level via keyboard. */
         step: 20,
 
-        template: [
-            '<div class="volume" role="application">',
-                '<p class="sr instructions" id="volume-instructions">',
+        template: HtmlUtils.joinHtml(
+            HtmlUtils.HTML('<div class="volume" role="application">'),
+                HtmlUtils.HTML('<p class="sr instructions" id="volume-instructions">'),
                     gettext('Click on this button to mute or unmute this video or press UP or DOWN buttons to increase or decrease volume level.'), // jshint ignore: line
```

Comment:
@robrap is it OK to use HTMLUtils.joinHtml in this way, where the translated strings aren't wrapped with anything?